ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.20-periodics.yaml

```diff
@@ -3219,18 +3219,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.20"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.20-amd64-nightly-aws-ipi-ovn-hypershift-guest-ext-oidc-f7
-  reporter_config:
```

Comment:
Slack notification is removed. Is it expected ?

Reply:
@liangxia good catch I missed. @xiuwang , `ci-operator/config/openshift/openshift-tests-private/.config.prowgen` has:
```
  - aws-ipi-ovn-hypershift-guest-ext-oidc-f7
  - aws-ipi-ovn-hypershift-guest-ext-oidc-f14
```
Now you changed the job's name, so aws-ipi-ovn-hypershift-guest-ext-oidc-f7 is removed from reporter_config here. You need update ci-operator/config/openshift/openshift-tests-private/.config.prowgen . My bad, didn't notice this, before.

Reply:
@xingxingxia @liangxia Updated, please have a look again, thanks